fog/ledger/enclave/api/src/messages.rs

```diff
@@ -1,13 +1,34 @@
 // Copyright (c) 2018-2021 The MobileCoin Foundation
 
 //! The message types used by the ledger_enclave_api.
-
-use fog_types::ledger::{CheckKeyImagesResponse, GetOutputsResponse};
+use crate::UntrustedKeyImageQueryResponse;
+use alloc::vec::Vec;
+use fog_types::ledger::GetOutputsResponse;
 use mc_attest_core::{Quote, Report, TargetInfo, VerificationReport};
 use mc_attest_enclave_api::{ClientAuthRequest, ClientSession, EnclaveMessage};
 use mc_common::ResponderId;
 use serde::{Deserialize, Serialize};
 
+// A struct representing the key image stores data
+#[derive(
+    Serialize,
+    Deserialize,
+    Debug,
+    Clone,
+    PartialEq,
+    PartialOrd,
+    core::cmp::Eq,
+    core::hash::Hash,
+    Ord,
+)]
+
+/// struct used when adding key image data using oram
```

Comment:
This looks odd to me, I don't think there should be new limes and comments between the derive part and the struct itself

Reply:
done